packages/core/src/components/Drawer/Drawer.stories.mdx

```diff
@@ -6,7 +6,7 @@ import * as stories from './Drawer.stories';
 
 # Drawer
 
-Drawer consists of three sections Header, Content and Footer. All of these you can directly use from Drawer component only like below ...
+A `Drawer` component provides access to locations in your application. It is anchored to the left or right edge of the screen. It consists of a Header, Content, and Footer. 
```

Comment:
`A Drawer component provides access to locations in your application.` This is not correct.